xcell/mappers/__init__.py

```diff
@@ -23,7 +24,7 @@
 from .mapper_CatWISE import MapperCatWISE
 from .mapper_ROSAT import MapperROSATXray
 from .mapper_dummy import MapperDummy
-from .utils import get_map_from_points, get_DIR_Nz, get_beam
+from .utils import get_map_from_points, get_DIR_Nz, get_beam, get_wf
```

Comment:
ok, since there's all kinds of window functions, can we be a bit more specific and call this `get_pixwin` or something like that?

Reply:
Actually, let's just hack this out tomorrow.